r/R/rzeppelin/R/common.R

```diff
@@ -0,0 +1,14 @@
+strintrplt <- function(snippet,envir=parent.frame()) {
```

Comment:
I'm confused, could you please explain a bit - this file looks very different from https://github.com/cran/rscala/blob/master/R/common.R - am I looking in the right place?